src/drivers/clock/riscv_clk/riscv_clk.c

```diff
@@ -38,7 +38,7 @@ static int clock_handler(unsigned int irq_nr, void *dev_id) {
 	(void)a6;
 	asm volatile("ecall");
 #else
-	clint_set_mtimecmp(clint_get_mtime() + COUNT_OFFSET);
+	clint_set_mtimecmp(clint_get_mtime() + COUNT_OFFSET, 0);
```

Comment:
I recommend that use `cpu_get_id()` instead of  0. Since `cpu_get_id()`  will just return 0. But this will increase compatibility

Reply:
@sycamoremoon thanks for suggestion, done with changes.